generic/Emscripten-wasm.cmake

```diff
@@ -49,7 +53,12 @@ set(CMAKE_SYSTEM_PREFIX_PATH ${CMAKE_FIND_ROOT_PATH})
 # compiler and linker. The *_INIT variables are available since CMake 3.7, so
 # it won't work in earlier versions. Sorry.
 cmake_minimum_required(VERSION 3.7)
-set(CMAKE_CXX_FLAGS_INIT "-s WASM=1")
-set(CMAKE_EXE_LINKER_FLAGS_INIT "-s WASM=1")
-set(CMAKE_CXX_FLAGS_RELEASE_INIT "-DNDEBUG -O3")
-set(CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT "-O3 --llvm-lto 1")
+
+string(APPEND CMAKE_CXX_FLAGS_INIT " -s WASM=1")
+string(APPEND CMAKE_CXX_FLAGS_INIT " -s DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR=1")
```

Comment:
This can break user code that relies on the old behavior and thus should be set explicitly only by the user. 

Actually, together with `POLYFILL_OLD_MATH_FUNCTIONS` to me it looks this PR contains a bunch of your private changes that you maybe didn't want to push upstream? :)